lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/aten_xla_type.cpp

```diff
@@ -622,21 +622,6 @@ at::Tensor& AtenXlaType::ne_(at::Tensor& self, const at::Tensor& other) {
   return self;
 }
 
-at::Tensor& AtenXlaType::normal_(at::Tensor& self, double mean, double std,
```

Comment:
A bit of both. It's not useful in practice, since these calls are only used to initialize weights. Also, I think we'd need to implement an additional operation to use our `normal_` lowering and I didn't want to spend time on it yet.